Sources/Utils/Utils.swift

```diff
@@ -128,7 +126,7 @@ class Utils {
     }
     
     static func getConditionString<T: Encodable>(conditions: T) -> String {
-        if let jsonData = try? self.jsonEncoder.encode(conditions), let jsonString = String(data: jsonData, encoding: .utf8) {
+        if let jsonData = try? JSONEncoder().encode(conditions), let jsonString = String(data: jsonData, encoding: .utf8) {
```

Comment:
This is expensive to initialize JSONEncoder on everycall, I think you may keep `static` inside the func. @jaeopt do you have any suggestion?

Reply:
declaring static property inside a method throws this error: `Static properties may only be declared on a type`, solution to this is declaring `private static let jsonEncoder = JSONEncoder()` inside the class like i had implemented earlier.

Reply:
declaring static property inside a method throws this error: `Static properties may only be declared on a type`, Workaround to this is declaring the property inside the class, similar to how it was done in the first commit.